ocaml/xapi/repository_helpers.ml

```diff
@@ -1606,3 +1606,44 @@ let update_livepatch_failures ~__context ~host ~applied ~failed =
   List.iter host_remove
     (List.filter_map Guidance.to_pending_guidance to_be_removed) ;
   List.iter host_add (List.filter_map Guidance.to_pending_guidance to_be_added)
+
+let assert_no_host_pending_mandatory_guidance ~__context ~host =
+  match Db.Host.get_pending_guidances ~__context ~self:host with
+  | [] ->
+      ()
+  | _ :: _ ->
```

Comment:
The `_ :: _` equals `_` here.

Reply:
It's good to make it explicit, when the code changes it can avoid silly mistakes